wasm/src/main/java/com/dylibso/chicory/wasm/types/ImportSection.java

```diff
@@ -1,14 +1,26 @@
 package com.dylibso.chicory.wasm.types;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
 public class ImportSection extends Section {
-    private Import[] imports;
+    private final ArrayList<Import> imports;
 
     public ImportSection(Import[] imports) {
         super(SectionId.IMPORT);
-        this.imports = imports;
+        this.imports = new ArrayList<>(List.of(imports));
+    }
+
+    public int importCount() {
+        return imports.size();
+    }
+
+    public Import import_(int idx) {
```

Comment:
something like `importNr` ?
The trailing `_` looks like generated code 😅

Reply:
It's always tricky when you have a conflict with a Java keyword. I've used the trailing `_` convention for a long time, but I'd be open to something else, as long as it would be something that would make sense for any name (i.e. not situation-specific - in this case the method happens to take an `int`-index argument but choosing a suffix on this basis means that a solution for this situation is not applicable to other situations).

Reply:
Naming is hard 😅 I don't think we will ever find a "general" solution that doesn't look like generated code.

Since this code is handwritten I would be willing to keep underscores as a last resource, what do you think about `importIdx`? Similar signatures are already used in the codebase and it will be nice to streamline more.

I don't want to drag this PR long for this, nitpicky detail, but it would be great to find a good naming, even if it's situation-specific, in my opinion. wdyt?

Reply:
Should we then rename all of the other section getters to `xxxIdx` as well?

Reply:
I don't like it, honestly; it doesn't read well. To read properly it would have to be something like `importForIndex` or something like that, but now it's a little strange because `import` is a verb so it sounds like something is supposed to happen when you call it. And, it's a bit wordy. What about using `getXxx` uniformly when there is an integer index, if `import_` is unacceptable?

Reply:
Make sense, I don't have super strong opinions here, I let someone else chip in, lgtm either way, thanks for trying things out! :+1:

Reply:
I don't like to keep this PR floating around for such a small detail, @bhelx and @danielperano can chip into this conversation later on and I'll take the action of fixing on top when necessary.
@dmlloyd sorry for making you wait on this one, if you can resolve the conflict it's ready to merge for me.